### ISSUE TITLE
Cannot load multiple TerrainControl worlds w/Sponge, 1.10

### ISSUE THREAD

**Issue by mcsnetworks** [MCTCP/TerrainControl#447]
Hello,

So I got one world loaded into TerrainControl but when I go to create the second world I get the following error:

http://pastebin.com/raw/evkPeKAW

Let me know if you need additional information.

**Comment by mcsnetworks** [comment]
This error is also easy to re-produce.

**Comment by mcsnetworks** [comment]
I think you'll want this pastie as well.  It appears as if the server is trying to load it twice?

http://pastebin.com/raw/u9EqAs6D

**Comment by rutgerkok** [comment]
Yeah, this is a missing feature. The Forge version is not able to load multiple worlds at the same time yet. It would be nice if it could.

**Comment by mcsnetworks** [comment]
Rutgerkok,  I had 12 worlds loaded under Cauldron.  What exactly is missing with Sponge/Forge that doesn't allow it.  This is a limitation I'm not aware of.

**Comment by mcsnetworks** [comment]
Rutgerkok,

What exactly is missing to support this?  I kinda need this in order to move my server from Cauldron 1.7.10 to SpongeForge 1.10.

Thanks,

**Comment by mcsnetworks** [comment]
Rutgerkok, 

Do you hide out on any IRC, I'd like to PM you regarding this issue.

Thanks,

**Comment by mcsnetworks** [comment]
@rutgerkok , I found what I think is a fix that will work for us,  basically I remove the following if check:

https://github.com/MCTCP/TerrainControl/blob/master/platforms/forge/src/main/java/com/khorn/terraincontrol/forge/WorldLoader.java#L144

Once I removed that line I was able to load and create multiple TC worlds within Forge without issue.  On our server we don't ever unload a world once its loaded so I won't run into a double load scenario since we only load the world once per server session.

**Comment by BakermanLP** [comment]
@mcsnetworks Nice to see someone fixed that issue. I tried your solution, but this doesn't seem to work for me. Can you please explain the following to me:
- How do you create new dimensons / worlds? 
  - What mod / sponge pluging
  - What commands
- Can you please give me your patched and compiled binary 

Thank you

Michael (aka BakermanLP)

**Comment by mcsnetworks** [comment]
My fix did not work.  TerrainControl for Forge doesn't load unique instances of each world.  Therefore each additional world overwrites the ForgeWorld configs that are loaded, thus screwing everything up.

I've been trying to get @rutgerkok to finish this and even volunteered to pay for his time but have received no response.

**Comment by mcsnetworks** [comment]
We are testing a few changes internally to add Multi-World support for TC Forge.  I'll PR it once we finalize the code.

**Comment by Meronat** [comment]
I believe this was solved with https://github.com/MCTCP/TerrainControl/pull/469 and https://github.com/MCTCP/TerrainControl/commit/ab469d50a90776ebe939cf418819c4e67cedcda2. Can be closed.
